### PR TITLE
Fixed some csv reading types

### PR DESCRIPTION
**Pull request by Cali0707**
Fixes enums in the csv to match the spec

### CODE REVIEW

**Comment by anant00729** [comment]
This works for me. @smattingly please check.

**Review by anant00729 — APPROVED** [review]

**Review by smattingly — CHANGES_REQUESTED** [review]
It looks like sensorController is still using the old strings.

File: packages/api/src/controllers/sensorController.js [hunk 1]
```diff
@@ -102,10 +102,23 @@ const sensorController = {
...
+            }
+            const allowedReadingTypes = [
+              'soil_water_content',
+              'soil_water_potential',
+              'temperature',
+            ];
+            let isValid = true;
+            val.forEach((readingType) => {
+              if (!allowedReadingTypes.includes(readingType)) {
+                isValid = false;
```
Comment (smattingly) [inline packages/api/src/controllers/sensorController.js:117]: Suggestion: return false here, return true on line 120, and eliminate the variable.

**Review by smattingly — CHANGES_REQUESTED** [review]

File: packages/api/src/controllers/sensorController.js (hunk 1 above)
Comment (Cali0707) [reply packages/api/src/controllers/sensorController.js]: Fixed

**Review by smattingly — APPROVED** [review]

**Pull request merged**